### DISCUSSION TITLE
How to optimise Pi 4 for 9MH/s?

### DISCUSSION THREAD

**Discussion by kicomoco** [revoxhere/duino-coin#1340] [Q&A]
I saw on the latest news in my duino coin web wallet that pi 4 should be able to get to 9MH/s. Mine (4gb) is only achieving 1.14MH/s with a clock speed of 1.7GHz.

What needs to be done to get to the 9MH/s speed?

Thanks

**Comment by revoxhere** [comment]
Make sure you're using fasthash accelerations and a 64 bit OS.

**Reply by clownfish81** [reply]
How are you getting 9MH/S on a raspberry pi while I'm only getting 1MH/S on low difficulty on a MacBook ???????/

**Reply by Grantrocks** [reply]
How would you compile fasthash?
I tried to do it on rpi4 overclocked o 2ghz running official 64-bit os. It still says unsupported os.

**Comment by Grantrocks** [comment]
Figured it out. Go to web wiki and follow instructions to compile fasthash. Once fasthash is compiled run the pc miner from the fasthash folder.

**Reply by nerudesu** [reply]
Might also link to the article for future reference: [How to compile fasthash accelerations](https://github.com/revoxhere/duino-coin/wiki/How-to-compile-fasthash-accelerations)

**Comment by yaroslavverstiuk** [comment]
I am reaching almost 13MH/s on Raspberry PI 4, and I think it can be higher 
Its overclocked to 2.3 and more optimized  custom mining script, I am planning to setup better cooling for it

<img width="855" alt="Screenshot 2023-01-06 at 3 22 31 PM" src="https://user-images.githubusercontent.com/48433659/211094131-08c45aae-91c3-4f09-9d3d-2958f43d72c7.png">

**Reply by Grantrocks** [reply]
Nice job! I was able to get my pi 4 overclocked at 2.2 to a max of 11.8MH/s.

**Reply by AMsGthub** [reply]
how much duco per day you can get
